File: xml/lua/read/xml_read.lua
-- ----------------------------------------------------
--	xml_read.lua
--
--				May/27/2011
--
-- ----------------------------------------------------
require	('xml_manipulate')
require	('file_io')

print ("*** 開始 ***")

file_xml=arg[1]

str_xml=file_to_str_proc (file_xml)

xml_string_display (str_xml)


print ("*** 終了 ***")
-- ----------------------------------------------------

File: xml/ocaml/read/xml_read.ml
(* ------------------------------------------------------------------- *)
(*
	xml_read.ml

					Feb/09/2012

open	Xml
*)
(* ------------------------------------------------------------------- *)
open	Unix

open	Text_manipulate
open	Xml_manipulate
(* ------------------------------------------------------------------- *)
let _ = print_endline "*** 開始 ***"

let file_xml = Sys.argv.(1)

let str_xml = file_to_str_proc file_xml

let _ = show_xml_str_proc str_xml

let _ = print_endline "*** 終了 ***"
(* ------------------------------------------------------------------- *)
